.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Issue
+
+What issue is this PR targeting: link here
```

Comment:
`What issue is this PR targeting? Is there no issue that covers the problem addressed here? Please open a corresponding issue and link it from here.`